x/qgb/keeper/keeper_valset_test.go

```diff
@@ -170,3 +170,34 @@ func TestCheckingAttestationNonceInValsets(t *testing.T) {
 		})
 	}
 }
+
+func TestEVMAddresses(t *testing.T) {
```

Comment:
Does it make sense to also add a test where there is a duplicate EVM address and see the create validator fail?

Reply:
Yeah that shouldn't be too difficult to do. Let me give it a go

Reply:
[add test for handling collision](https://github.com/celestiaorg/celestia-app/pull/2169/commits/fc5388b9dc1f3e7681d9eb5b876370b58b9002b4)

Reply:
:+1: